src/messages/en.json

```diff
@@ -36,5 +37,16 @@
   "loginTab": {
     "Forgot your password?": "Forgot your password?",
     "Don't have an account? Sign up": "Don't have an account? Sign up"
+  },
+  "gamePage": {
+    "Genres": "Genres",
+    "Features": "Features",
+    "BUY NOW": "BUY NOW",
+    "ADD TO CART": "ADD TO CART",
+    "ADD TO WISHLIST": "ADD TO WISHLIST",
```

Comment:
tutaj trzymajmy się jednej konwencji jeśli to normalne tłumaczenie to zostawmy pisane normalnie nie capslockiem całość